Contributions.md

```diff
@@ -0,0 +1,9 @@
+# Contributions
+
+## Project: RCL RHours Accounting
+
+Begin | End | Contributor | Amount | Claims
+----- | --- | ----------- | ------ | ------
+2018-05-01 | 2018-08-21 | glenbraun | 1.00 Hours | Set up RHours Accounting
+2018-05-01 | 2018-08-21 | glenbraun | 0.75 Hours | Updated README with policies; Created contributions report; Time split with RHours project
+2018-05-01 | 2018-08-20 | Jake-Gillberg | 85.00 Hours | Founded RCL.
```

Comment:
I don't see an item here for the License. Also, I think I'm fine with just showing hours right now, because I think having a $50/hr policy across the board is pretty good.